### DISCUSSION TITLE
腳本無法更新

### DISCUSSION THREAD

**Discussion by 23bebe** [whyour/qinglong#1811] [Q&A]
ql repo 運行會提示

任务未运行或运行失败，请尝试手动运行

設備是R4S ，docker

**Comment by whyour** [comment]
> ql repo 運行會提示
> 
> 
> 
> 任务未运行或运行失败，请尝试手动运行
> 
> 
> 
> 設備是R4S ，docker

移到订阅管理中添加

**Comment by 23bebe** [comment]
谢谢大佬

在 2023年2月14日週二 22:36，whyour ***@***.***> 寫道：
